lib/lazy_api_doc/generator.rb

```diff
@@ -45,15 +45,22 @@ def example_group(example, examples, route)
           "summary" => example.action,
           "parameters" => path_params(route, examples) + query_params(route, examples),
           "requestBody" => body_params(route, examples),
-          "responses" => examples.group_by { |ex| ex.response['code'] }.transform_values do |variants|
-            {
-              "description" => variants.first["description"].capitalize,
+          "responses" => examples.group_by { |ex| ex.response['code'] }.map do |code, variants|
```

Comment:
responses should be a hash but not an array as I remember. Are you sure that `map` is a proper method here?